librz/include/rz_types_base.h

```diff
@@ -37,7 +37,7 @@ typedef struct _ut96 {
 } ut96;
 typedef struct _ut128 {
 	ut64 Low;
-	st64 High;
+	ut64 High;
```

Comment:
@XVilka @wargio any idea why it was `st64`?

Reply:
no idea. but makes no sense.

Reply:
No idea, was likely a typo.